api/src/main/java/io/opentelemetry/trace/DefaultTracer.java

```diff
@@ -148,7 +142,6 @@ public NoopSpanBuilder setStartTimestamp(long startTimestamp) {
     }
 
     private NoopSpanBuilder(Tracer tracer, String name) {
-      Utils.checkNotNull(tracer, "tracer");
```

Comment:
Why is this check removed?

Reply:
We always call it with "this" because NoopSpanBuilder is an internal class (nobody else will call it) so I don't need to check that :)